docs/source/activelearning.rst

```diff
@@ -2,3 +2,27 @@ Active Learning
 ~~~~~~~~~~~~~~~
 
 Introduction of Active Learning on (academic) literature.
+
+Active learning denotes the scenario in which a reviewer actively provides classifications for references which are presented by the model [3,4]. Subsequently, the model retrains based on the new input. When it has retrained, it decides on a new reference to be annotated and it shows this to the reviewer. In this way, the annotated dataset starts out small and iteratively grows in size [4]. In the case of Automated Systematic Review, this process is necessary to provide some initial classifications to the model, because the reviewer starts with a dataset without annotations.
```

Comment:
I think "the model" should be "the learning model" or "the machine learning model", or "the classifier". Am I miss something here? I don't have previous paragraphs.

Reply:
Agreed! Maybe call it "machine learning model" once and then refer to it as the model? Or keep referring to it as the "machine learning model"? Thanks for the feedback!

Reply:
I'd say calling it 'machine learning model' the first time and 'model' later on is fine!